src/clinical/clinical-service.ts

```diff
@@ -30,14 +31,16 @@ import {
   aliasEntityNames,
   allEntityNames,
 } from '../common-model/entities';
-import { Errors } from '../utils';
+import { Errors, notEmpty } from '../utils';
```

Comment:
`notEmpty` is unused

Reply:
Using now; sorry PR needed to be revised